### PR TITLE
fix: Fixes self service recovery and unlock account misalignments

### PR DESCRIPTION
**Pull request by manueltanzi-okta**
### Description

- Fixes self service recovery and unlock account misalignments

### Screenshots

Screen Shot 2019-07-10 at 9 51 02 AM
Screen Shot 2019-07-10 at 9 50 49 AM
Screen Shot 2019-07-10 at 9 50 37 AM
Screen Shot 2019-07-10 at 9 50 25 AM

### Reviewers

@robertdamphousse-okta @mauriciocastillosilva-okta @haishengwu-okta  @nikhilvenkatraman-okta @haoyuli-okta

### CODE REVIEW

File: src/RecoveryChallengeController.js [hunk 1]
```diff
@@ -73,7 +73,7 @@ function (Okta, FormController, FormType, Enums, FooterSignout, TextBox) {
           FormType.Button({
             title: Okta.loc('mfa.resendCode', 'login'),
             attributes: { 'data-se': 'resend-button' },
-            className: 'button sms-request-button',
+            className: 'button sms-request-button margin-top-30',
```
Comment (haoyuli-okta) [inline src/RecoveryChallengeController.js]: this also fixes phone call button i believe

File: assets/sass/modules/_forgot-password.scss [hunk 2]
```diff
@@ -35,7 +35,7 @@
     }
   }
 
-  .o-form-input {
```
Comment (mauriciocastillosilva-okta) [inline assets/sass/modules/_forgot-password.scss]: we don't need this for `.o-form-input` anymore?

File: assets/sass/modules/_forgot-password.scss [hunk 3]
```diff
@@ -35,7 +35,7 @@
...
+  .o-form-fieldset.enroll-sms-phone {
```
Comment (mauriciocastillosilva-okta) [inline assets/sass/modules/_forgot-password.scss]: what about `enroll-call-phone`? we need to fix that too

File: assets/sass/modules/_forgot-password.scss (hunk 2 above)
Comment (manueltanzi-okta) [reply assets/sass/modules/_forgot-password.scss]: no, we need this style in the external container only which includes also the label

File: assets/sass/modules/_forgot-password.scss (hunk 3 above)
Comment (manueltanzi-okta) [reply assets/sass/modules/_forgot-password.scss]: They use the same element with class `enroll-sms-phone`, is a unique controller, see the screenshots.

Comment (mauriciocastillosilva-okta) [reply assets/sass/modules/_forgot-password.scss]: ok...  And this is under `recovery-challenge` so all good  👍

**Pull request merged**